templates/blog/blog-card.html

```diff
@@ -1,6 +1,6 @@
 <div class="col-4 blog-p-card--post">
   {% if article.group %}
-  <header class="blog-p-card__header blog-p-card__header--{{ article.group.slug }}">
+  <header class="blog-p-card__header--{{ article.group.slug }}">
```

Comment:
Couldn't we add some logic here. If there is a group slug then `--{{ article.group.slug }}` is prepended to the class?

Reply:
Good point - updating that now